Enterprise-CMCS/eAPD
# Tbolt/4413 key state personnel mmis

Resolves #4413 

### Description
Updates to HITECH and MMIS apds to align with new requirements / designs

### Significant changes or possible side effects
MMIS Changes
* In key personnel - 3rd list item of the description says "Key personnel costs entered here are represented in the Proposed Budget, at the FFP specified below."
* Example box for MMIS says "Examples: Project/Product Manager, Development Lead, Business Owner"
* Added an abridged Federal State Split to the Key Personnel form
* Added validation for the Federal State Split to say "Select a federal-state split"
* Added new Medicaid Share % field
* The schema has been updated to save the fed-state split and medicaid share %
* Key State Personnel read-only component has been updated to display the fed-state split and medicaid 

Other
* In both HITECH and MMIS the key state personnel subform adds the hint text "Cost with benefits x FTE = Total" underneath the total


### Steps to manually verify this change

1. Login and create a new MMIS APD
2. Verify above changes are present. Including the "Other" ones that have been updated for both
3. Login and create a new HITECH APD
4. Verify above changes are present

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
